.circleci/integration-tests/master_dag.py

```diff
@@ -89,7 +90,7 @@ def prepare_dag_dependency(task_info, execution_time):
 
 with DAG(
     dag_id="example_master_dag",
-    schedule_interval="@daily",
+    schedule_interval=MASTER_DAG_SCHEDULE,
```

Comment:
Should we replace `schedule_interval` with `schedule`?

Reply:
+1 for this. maybe in this PR or in separate PR both ok to me

Reply:
not only here but also other places if we have this param

Reply:
a separate PR is ok to me

Reply:
https://github.com/astronomer/astronomer-providers/pull/1010

Reply:
shall we make the default value a cron expression? That way I feel both the env value and default value would be consistent, otherwise  `@daily` in 2 env we can't do?

Reply:
I don't have a strong opinion on this one. Either way works for me. Just updated it